tests/test_isrTaskLSST.py

```diff
@@ -92,6 +90,7 @@ def test_isrBootstrapBias(self):
         mock_config = self.get_mock_config_no_signal()
 
         mock = isrMockLSST.IsrMockLSST(config=mock_config)
+        mock_config.doAddSerialCti = False
```

Comment:
Is there a reason this has to be `False`?  I would think that either we have it treated similarly to BF (false by default in the mock config) or like the defects (true in the mock config and in the no_signal case).